<?xml version="1.0"?>

<launch>

  <!-- SINGABOAT-VRX Wayfinding -->
  <node pkg="singaboat_vrx" type="singaboat_wayfinding.py" name="singaboat_wayfinding" output="screen"/>

  <!-- SINGABOAT-VRX Inverse Kinematics -->
  <node pkg="singaboat_vrx" type="singaboat_inverse_kinematics.py" name="singaboat_inverse_kinematics" output="screen"/>

</launch>
